### PR TITLE
Create RELEASE.md

### PR DESCRIPTION
**Pull request by sherry-x**
Create a short description for Aptos release process

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+```
+
+### main branch
+All current development occurs on the `main` branch. All new feature development
+should use feature flag to gate it off during development, and turn it on once the
+development is complete and passed the AIP process when it applies.
+
+### devnet branch
+The `devnet` branch is created of the `main` branch every week. It will be used to deploy
```
Comment (movekevin) [inline RELEASE.md:25]: Why don't we use tag here? Similar to how we can use mainnet/testnet/mainnet-framework/testnet-framework tags

Comment (sherry-x) [reply RELEASE.md:25]: It would likely be very spammy in our release page if we start creating devnet tags for every devnet release.

Comment (sherry-x) [reply RELEASE.md:25]: Also note that tags can't be moved very easily, if you created a tag on a specific commit, it's somewhat forever. So in github, arbitrary tag won't work, we can't just have a `devnet` tag, and move it whenever we cut new commit. every tag should be unique.

Comment (movekevin) [reply RELEASE.md:25]: I see

**Review by movekevin — APPROVED** [review]

**Pull request merged**